.circleci/config.yml

```diff
@@ -136,10 +136,11 @@ jobs:
             gotestsum --junitfile "$TEST_RESULTS/gotestsum-report.xml" -- ./... -p 1 -timeout 20m -failfast \
               -enable-enterprise \
               -enable-multi-cluster \
+              -run TestController \
               -kubeconfig="$primary_kubeconfig" \
               -secondary-kubeconfig="$secondary_kubeconfig" \
               -debug-directory="$TEST_RESULTS/debug" \
-              -consul-k8s-image=hashicorpdev/consul-k8s:latest
+              -consul-k8s-image=hashicorpdev/consul-k8s:crd-controller-base-latest
```

Comment:
This will be updated once https://github.com/hashicorp/consul-k8s/pull/353 is merged